package.json

```diff
@@ -72,6 +72,7 @@
         "react-social-login-buttons": "^3.6.1",
         "react-stay-scrolled": "^8.0.0",
         "react-use": "^17.4.0",
+        "readable-numbers": "^1.0.7",
```

Comment:
Used to format long numbers into a short human readable format to save space in the table